components/eam/src/physics/cam/zm_conv.F90

```diff
@@ -72,12 +72,12 @@ module zm_conv
    real(r8) :: zmconv_auto_fac       = unset_r8
    real(r8) :: zmconv_accr_fac       = unset_r8
    real(r8) :: zmconv_micro_dcs      = unset_r8
-
+!++ MCSP
```

Comment:
Why are you adding more code tags in? These are completely redundant since all the variables enclosed in the tags already have "MCSP" in the name! Similar logic follows for sections below. The code tags don't make the code more clearer or easier to grep through, so why add them?

Reply:
I'd removed them in a previous code clean up, but these older tags must've crept in when I copied over the fixes for ZM microphysics. Will remove them.